wasm/CMakeLists.txt

```diff
@@ -61,16 +61,16 @@ add_test(NAME elastix-wasm-2d-test
     -m ${CMAKE_CURRENT_SOURCE_DIR}/test/data/input/CT_2D_head_moving.iwi.cbor
   )
 
-add_test(NAME elastix-wasm-2d-initial-transform-test
-  COMMAND elastix
-    ${CMAKE_CURRENT_SOURCE_DIR}/test/data/input/parameters_single.json
-    ${CMAKE_CURRENT_BINARY_DIR}/CT_2D_head_registered_initial.iwi.cbor
-    ${CMAKE_CURRENT_BINARY_DIR}/CT_2D_head_initial.h5
-    ${CMAKE_CURRENT_BINARY_DIR}/CT_2D_head_initial.json
-    -f ${CMAKE_CURRENT_SOURCE_DIR}/test/data/input/CT_2D_head_fixed.iwi.cbor
-    -m ${CMAKE_CURRENT_SOURCE_DIR}/test/data/input/CT_2D_head_moving.iwi.cbor
-    -i ${CMAKE_CURRENT_SOURCE_DIR}/test/data/input/CT_2D_head_translation.h5
-  )
+#add_test(NAME elastix-wasm-2d-initial-transform-test
+  #COMMAND elastix
+    #${CMAKE_CURRENT_SOURCE_DIR}/test/data/input/parameters_single.json
+    #${CMAKE_CURRENT_BINARY_DIR}/CT_2D_head_registered_initial.iwi.cbor
+    #${CMAKE_CURRENT_BINARY_DIR}/CT_2D_head_initial.h5
+    #${CMAKE_CURRENT_BINARY_DIR}/CT_2D_head_initial.json
+    #-f ${CMAKE_CURRENT_SOURCE_DIR}/test/data/input/CT_2D_head_fixed.iwi.cbor
+    #-m ${CMAKE_CURRENT_SOURCE_DIR}/test/data/input/CT_2D_head_moving.iwi.cbor
+    #-i ${CMAKE_CURRENT_SOURCE_DIR}/test/data/input/CT_2D_head_translation.h5
+  #)
```

Comment:
I guess this code is only _temporarily_ commented out, right? (Otherwise I would suggest to just remove the code. It will be restorable from git any day anyway.)

Reply:
Yes, I would like to keep it here for visibility. I hope it will be fixed with the next toolchain update.